src/mesh/grids/nonsymmetric_curved_periodic_grid.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef __NONSYMMETRIC_CURVED_PERIODIC_GRID_H__
+#define __NONSYMMETRIC_CURVED_PERIODIC_GRID_H__
+
+#include <deal.II/grid/manifold_lib.h>
+#include <deal.II/distributed/tria.h>
+
+namespace PHiLiP {
+namespace Grids {
+
+/// Create a Gaussian bump grid with an associated nonlinear manifold.
```

Comment:
Update description. The class description is a good place to put the reference for the grid.

Reply:
fixed